ansible/playbooks/monitoring.yml

```diff
@@ -62,6 +62,7 @@
               - docsrs.infra.rust-lang.org:9100
               - bastion.infra.rust-lang.org:9100
               - play-1.infra.rust-lang.org:9100
+              - play-2.infra.rust-lang.org:9100
```

Comment:
Let's actually get monitoring up and running first, before switching the primary instance. Otherwise, I may be scrambling to understand what's going on when we switch.

Reply:
https://github.com/rust-lang/simpleinfra/pull/587